src/catalogue/category/cataloguePropertiesForm.component.test.tsx

```diff
@@ -49,6 +50,9 @@ describe('Catalogue Properties Form', () => {
   });
 
   it('renders correctly', async () => {
+    vi.mock('uuid', () => {
+      return { v4: vi.fn(() => '00000000-0000-0000-0000-000000000000') };
```

Comment:
Could you use something like this in `testUtils` so that they don't all use the same ids.
```
/* This counter and the function below it are used for forcing UUID's to be deterministic */
let globalUUIDCounter = 0;

const generateUUIDBytes = (): Uint8Array => {
  const byteArray = new Uint8Array(16);
  for (let i = 0; i < byteArray.length; i++) {
    byteArray[i] = (globalUUIDCounter + i) % 256; // Ensuring the value stays within byte range
  }
  globalUUIDCounter++;
  return byteArray;
};

const mockUUIDv4 = () => {
    // Mock uuid generation in all tests to ensure they are deterministic rather than random
  // for snapshots
  globalUUIDCounter = 0;
  vi.mock('uuid', async () => {
    const uuid = await vi.importActual('uuid');
    return {
      ...uuid,
      v4: vi.fn(() =>
        // @ts-expect-error Unknown as don't want to import here
        uuid.v4({
          random: generateUUIDBytes(),
        })
      ),
    };
  });
}
```